### PR TITLE
Fix GitHub workflows

### PR DESCRIPTION
**Pull request by Slach**

### CODE REVIEW

File: .github/workflows/run_tests.yaml [hunk 1]
```diff
@@ -6,7 +6,25 @@ jobs:
...
     steps:
       - uses: actions/checkout@v2
+      - run: |
+          sudo apt-get update
+          sudo apt-get install -y conntrack
+      - uses: medyagh/setup-minikube@latest
+        with:
+          driver: docker
+          container-runtime: containerd
+          kubernetes-version: v1.21.9
```
Comment (zvonand) [inline .github/workflows/run_tests.yaml:16]: why not 1.23?

File: tests/image/Dockerfile [hunk 2]
```diff
@@ -79,7 +79,7 @@ EXPOSE 2375
 
 ENV MINIKUBE_HOME /home/master/.minikube
 
-ENV K8S_VERSION=${K8S_VERSION:-1.22.2}
+ENV K8S_VERSION=${K8S_VERSION:-1.21.6}
```
Comment (zvonand) [inline tests/image/Dockerfile:82]: Maybe we should use the same k8s version all way long to keep things simpler?

File: .github/workflows/run_tests.yaml (hunk 1 above)
Comment (Slach) [reply .github/workflows/run_tests.yaml:16]: potentially current prometheus-operator and grafana-operator not works on 1.22+, we need upgrade it first in separate PR it need research, for example grafana-operator 4.x version broke installation process compatible with 3.9.x

File: tests/image/Dockerfile (hunk 2 above)
Comment (Slach) [reply tests/image/Dockerfile:82]: 1,21,6 it's last 1.21 version available on minikube preloaded archives

**Review by zvonand — APPROVED** [review]
👍🏻

**Pull request merged**